docs/COMPATIBILITY.md

```diff
@@ -262,6 +268,14 @@ Note that MS SQL Server and Oracle are not supported as database backends.
 		<th><a href="https://github.com/garlik/4store">4store</a></th>
 		<th><a href="http://rdf4j.org/">Sesame</a></th>
 		<th><a href="https://wiki.blazegraph.com/">Blazegraph</a></th>
+	</tr>
+		<tr>
+		<th>SMW 3.0.x</th>
+		<td>Full support<br />(1.x >=1.1) + 2.4.0</td>
+		<td>Full support<br />(6.x >=6.1) + 7.2<sup>[t.1]</sup></td>
+		<td>Beta support<br />(1.x >=1.1)<sup>[t.2]</sup></td>
+		<td>Full support<br />(2.8.x)</td>
+		<td>Full support<br />(1.5.2) + 2.1.0<sup>[t.3]</sup></td>
```

Comment:
@mwjames Here I assumed that there are no compat changes with the rdf stores.

Reply:
Remained the same.